### PR TITLE
add new mouse apis and update changelog

### PR DESCRIPTION
**Pull request by DiceyClickcorp**

### CODE REVIEW

File: doc/references/python/changelog/changelog.md [hunk 1]
```diff
@@ -1,4 +1,19 @@
 # Change Log
+## 0.1.5 (26 August 2022)
+### New features
+- New api: support mouse emulation events, clicknium.mouse.click/double_click/up/down/move/scroll/positon.
```
Comment (kayYOLO) [inline doc/references/python/changelog/changelog.md:4]: API

Comment (DiceyClickcorp) [reply doc/references/python/changelog/changelog.md:4]: won't fix now

File: doc/references/python/changelog/changelog.md [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+- Support turning on browser extensions for chrome and edge automatically.
```
Comment (kayYOLO) [inline doc/references/python/changelog/changelog.md:5]: Chrome and Edge

Comment (DiceyClickcorp) [reply doc/references/python/changelog/changelog.md]: done

**Review by LouisHQ — APPROVED** [review]

File: doc/references/python/changelog/changelog.md (hunk 1 above)
Comment (automation9417) [inline doc/references/python/changelog/changelog.md:4]: position

Comment (yongjuntian) [inline doc/references/python/changelog/changelog.md:4]: position

File: doc/references/python/changelog/changelog.md [hunk 3]
```diff
@@ -1,4 +1,19 @@
 # Change Log
+## 0.1.5 (26 August 2022)
+### New features
+- New api: support mouse emulation events, clicknium.mouse.click/double_click/up/down/move/scroll/positon.
+- Support turning on browser extensions for Chrome and Edge automatically.
```
Comment (yongjuntian) [inline doc/references/python/changelog/changelog.md:5]: Improve the experience of installing browser extensions

File: doc/references/python/changelog/changelog.md [hunk 4]
```diff
@@ -1,4 +1,19 @@
 # Change Log
+## 0.1.5 (26 August 2022)
+### New features
+- New api: support mouse emulation events, clicknium.mouse.click/double_click/up/down/move/scroll/positon.
+- Support turning on browser extensions for Chrome and Edge automatically. 
+- Support validation elements with specified index.
+- Support attribute `name` for UIA locator.
```
Comment (yongjuntian) [inline doc/references/python/changelog/changelog.md:7]: move to Improvements,
Fix bug: missing 'name' attribute while recording for UIA locator

File: doc/references/python/changelog/changelog.md [hunk 5]
```diff
@@ -1,4 +1,19 @@
...
+
+### Improvements
+- Optimize the default names for locators.
+- Refactor JAVA automation, optimize automation on JAVA applications.
```
Comment (yongjuntian) [inline doc/references/python/changelog/changelog.md:11]: Refine Java automation and Java application automation

File: doc/references/python/mouse/down.md [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+# clicknium.mouse.down
+
+***def down(
+        self,
+        x: int, 
+        y: int, 
+        mouse_button: Literal["left", "middle", "right"] = MouseButton.Left
+    ) -> None***  
+
+Move the mouse cursor to the X and Y integer coordinates, and press the mouse button down.
```
Comment (yongjuntian) [inline doc/references/python/mouse/down.md:15]: Place the mouse at the desired location.

**Pull request merged**